jerry-core/api/jerry.c

```diff
@@ -846,10 +867,24 @@ jerry_value_set_error_flag (jerry_value_t *value_p)
 
   if (!ecma_is_value_error_reference (*value_p))
   {
-    *value_p = ecma_create_error_reference (*value_p);
+    *value_p = ecma_create_error_reference (*value_p, true);
   }
 } /* jerry_value_set_error_flag */
 
+/**
+ * Set both the abort and error flags if the value is not an error reference.
+ */
+void
+jerry_value_set_abort_flag (jerry_value_t *value_p)
+{
+  jerry_assert_api_available ();
+
+  if (!ecma_is_value_error_reference (*value_p))
```

Comment:
What happens if a reference error. Shouldn't we set the abort flag in such cases as well?

Reply:
It is possible, but the description says it ignores error references. Shall we change it?

Reply:
I would change it.